src/components/AddToCartButton/AddToCartButton.tsx

```diff
@@ -16,12 +16,14 @@ export interface AddToCartButtonProps {
   variant?: 'cart' | 'list',
   onClick: (wishlist?: string) => void;
   popoverData?: Array<{id: string, name: string;}>;
+  openCreateListPopup?: (e: Event) => void;
 }
 
 export const AddToCartButton: FunctionComponent<AddToCartButtonProps> = ({
   variant = 'cart',
   onClick,
   popoverData = [],
+  openCreateListPopup,
```

Comment:
Add a default callback function.

```suggestion
  openCreateListPopup = () => undefined,
```